db/migrate/20230128065801_createlike.rb

```diff
@@ -0,0 +1,3 @@
+class Createlike < ActiveRecord::Migration[7.0]
+  def change; end
```

Comment:
- [ ] It looks like you have not created the `Likes` table because I have checked the `schema.rb` file and I didn't find it. Please create the likes table and make sure it includes the reference to the author of the like and also the reference to the post just like you did with the `comments` table 😃 👍